src/libs/antares/study/parts/hydro/series.cpp

```diff
@@ -322,5 +303,31 @@ uint64 DataSeriesHydro::memoryUsage() const
     return sizeof(double) + ror.memoryUsage() + storage.memoryUsage();
 }
 
+void DataSeriesHydro::AdjustMonth(Study& study, uint firstDayMonth[13], uint daysPerMonthDecals[12])
```

Comment:
```suggestion
void DataSeriesHydro::AdjustMonth(const Study& study, uint firstDayMonth[13])
```

- `study` isn't changed
- `uint daysPerMonthDecals[12]` is just a temporary variable, you can declare it within the impl of `DataSeriesHydro::AdjustMonth`

Reply:
daysPerMonthDecals is used outside of the function, L 166 and 169